go.mod

```diff
@@ -3,65 +3,74 @@ module github.com/GoogleCloudPlatform/opentelemetry-operations-collector
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/activedirectorydsreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/apachereceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/couchdbreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/elasticsearchreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/flinkmetricsreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jmxreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/memcachedreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/nginxreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/postgresqlreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/rabbitmqreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/saphanareceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlqueryreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.64.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.64.0
 	github.com/shirou/gopsutil v3.21.10+incompatible
 	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector v0.63.1
-	go.opentelemetry.io/collector/exporter/loggingexporter v0.63.1
-	go.opentelemetry.io/collector/exporter/otlpexporter v0.63.1
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.63.1
-	go.opentelemetry.io/collector/pdata v0.63.1
-	go.opentelemetry.io/collector/semconv v0.63.1
+	go.opentelemetry.io/collector v0.64.1
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.64.1
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.64.1
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.64.1
+	go.opentelemetry.io/collector/extension/ballastextension v0.64.1
```

Comment:
Were this, `zpagesextension`, and `memorylimiterprocessor` brought in for a reason or was this an automated thing? If it was on purpose, I think we need to add them to `components.go` right? (I haven't heard anything about our plans with these but in fairness it would make sense for us to bring these in)

Reply:
I will double check. That wasn't intentional.

Reply:
Very weird, no idea how these were in `components.go` without being in `go.mod` all this time. Well at least it's correct now!